web/src/components/pages/profile/avatar-setup/avatar-setup.tsx

```diff
@@ -19,17 +19,16 @@ import {
   PlayIcon,
 } from '../../../ui/icons';
 import { Button } from '../../../ui/ui';
-import {
-  PlayLink,
-  RecordLink,
-  PrimaryButton,
-  Voice,
-} from '../../../primary-buttons/primary-buttons';
+import { Voice } from '../../../primary-buttons/primary-buttons';
 import AudioIOS from '../../contribution/speak/audio-ios';
 import AudioWeb, { AudioError } from '../../contribution/speak/audio-web';
 import { isFirefoxFocus, isNativeIOS, isProduction } from '../../../../utility';
+import { Suspense, lazy } from 'react';
+const Lottie = lazy(() => import('react-lottie'));
+const animationData = require('../../../layout/data.json');
 
 import './avatar-setup.css';
+import { string, object } from 'prop-types';
```

Comment:
Why do we import PropTypes here? Don't we do all the typing with TS here?

Reply:
Ah, maybe this is added mistakenly by me while editor was showing some options of auto-fix any problem

Reply:
Makes sense! Should be safe to remove